docs/dbt/variable-interpolation.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "dbt variable interpolation"
+sidebarTitle: "Variable interpolation"
+---
+## Overview
+
+Mage allows users to reference variables
+specific to your [pipeline](/getting-started/runtime-variable) or [project](/development/environment-variables). These variables are accessible throughout your dbt profiles, pipelines, and models, using the following syntax. 
+
+| Syntax                   | Description                     | Example                             |
```

Comment:
We also support other syntaxes: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/data_preparation/shared/utils.py#L7-L34
```
{{ mage_secret_var('...') }}
{{ aws_secret_var('...') }}
{{ azure_secret_var('...') }}
```
I would suggest to have a central doc to list all the variable syntax we support.

Reply:
I like this idea, I'll create a ticket

Reply:
can we leave this page for now, then link to the new one once it's up?